src/test/java/org/jabref/model/database/BibDatabaseContextTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jabref.model.database;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.metadata.FileDirectoryPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+public class BibDatabaseContextTest {
+
+    private String currentWorkingDir;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
As I understand, it "Returns a rule that expects no exception to be thrown" (http://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html). Although this is the default behaviour, it explicitly states that these tests MUST NOT through an exception, since this is what we test for. The same construct is used in BibDatabaseTest.java, and make IMHO the intent of the test more explicit and easier to read. I suggest leaving it.